Logging/VERSION

```diff
@@ -1 +1 @@
-1.12.2
\ No newline at end of file
+1.13.0
```

Comment:
I could see #1140 being a patch, since it is just clarifying the message

Reply:
Done